LoRaEngine/modules/LoRaBasicsStationModule/helper-functions.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
```

Comment:
I guess you also have to copy this file in the basicstation dockerfile?

Reply:
Done, thanks!